README.md

```diff
@@ -567,6 +567,105 @@ unsubscribe();
 
 You can remove all listeners at once by calling `reactions.unsubscribeAll()`.
 
+## Handling encoded objects in integrations
+
+If you have set up an [Ably integration](https://ably.com/docs/general/integrations) to receive events from your chat
+room, depending on your configuration, you may receive these as encoded objects.
+See [here](https://ably.com/docs/general/webhooks) for more information.
+Should you wish to convert this object to a chat type, you can use the functions below to help you.
+
+For example, if you have the following item reach your integration:
+
+```json
+{
+  "items": [
+    {
+      "webhookId": "Ja-tsg",
+      "source": "channel.message",
+      "serial": "108iZpUxQBe4Vv35120919@1720954404104-0",
+      "timestamp": 1720954404104,
+      "name": "channel.message",
+      "data": {
+        "channelId": "some-room::$chat::$chatMessages",
+        "site": "eu-west-1-A",
+        "messages": [
+          {
+            "id": "chat:6TP2sA:some-room:a4534b0ab37bdd5:0",
+            "clientId": "user1",
+            "timestamp": 1720954404104,
+            "encoding": "json",
+            "extras": {
+              "timeserial": "108iZpUxQBe4Vv35120919@1720954404104-0",
+              "headers": {}
+            },
+            "data": "{\"text\":\"some text data\",\"metadata\":{}}",
+            "name": "message.created"
+          }
+        ]
+      }
+    }
+  ]
+}
+```
+
+If you are unsure as to the underlying chat entity type of the encoded message,
```

Comment:
Should this be the default recommendation, that you should use `getEntityTypeFromEncoded`?

Reply:
Yeh I agree with you. I had thought it would depend on how you set up your integration, i.e. if you choose only to send reactions to your integration via channel rule filter, but by setting the default recommendation as always calling `getEntityTypeFromEncoded` you catch that case anyway :)